.github/workflows/cc-plugin-glider-test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         create-args: >-
           python=3 pip
           --file requirements.txt
-          --file test_requirements.txt
+          --file requirements-test.txt
```

Comment:
Changes like this make it hard to follow the actual changes in the PR. While I personally prefer the `requirements-dev.txt` we are stuck with the original file name here and no need to change it now.

PS: I'd welcome a change like this in its own PR, away from important code changes that require careful review.